setup.py

```diff
@@ -6,6 +6,66 @@ def requirements():
         return f.read()
 
 
+long_description = """
```

Comment:
Is there a possibility to read in selectively from the README.rst file? This might help with maintenance, especially if the README changes over time.

Reply:
I think that would be wise. How smart should it be? 

The simplest solution would be to provide something like pairs of line numbers from which to extract text.  That will need to be verified whenever the README is updated. 

The other I option I see is to scan the text for which sections to copy. We can easily find the text of the `Functionality` section, but there is no concrete marker of where we should stop for the first section.

Reply:
Or maybe the best solution is to add some comments to the README that will mark relevant sections

Reply:
I like the last suggestion you have - pragmatic and smart. Let’s go with that.